.travis.yml

```diff
@@ -2,21 +2,24 @@ sudo: false
 language: python
 matrix:
   include:
+    - python: 3.6
+      env: TOXENV=lint
     - python: 3.4
       env: TOXENV=py34
     - python: 3.5
       env: TOXENV=py35
     - python: 3.6
       env: TOXENV=py36
-    - python: 3.6-dev
-      env: TOXENV=py36-dev
-    - python: nightly
+    - python: 3.7
       env: TOXENV=py37
-    - python: 3.6
-      env: TOXENV=lint
+      dist: xenial
```

Comment:
Is `dist` necessary for some reason?

Reply:
Yes, see https://github.com/travis-ci/travis-ci/issues/9815

Reply:
Great, thanks! I guess at some point we can make a newer dist the default, but for now this is good.

Reply:
Yeah, the only reason I didn't do it here is that there is not a Python 3.4 package available for Xenial.